packages/@vue/cli-service/lib/config/css.js

```diff
@@ -234,9 +234,7 @@ module.exports = (api, rootOptions) => {
       ))
     }
     createCSSRule('less', /\.less$/, 'less-loader', loaderOptions.less)
-    createCSSRule('stylus', /\.styl(us)?$/, 'stylus-loader', Object.assign({
-      preferPathResolver: 'webpack'
```

Comment:
stylus-loader [v4](https://github.com/webpack-contrib/stylus-loader/releases/tag/v4.0.0), `preferPathResolver` option has been removed. `stylus` options were moved to `stylusOptions`.